test/azure/legacy/Expected/AcceptanceTests/CustomBaseUri/generated_samples/paths_get_empty.py

```diff
@@ -0,0 +1,30 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+
+from custombaseurl import AutoRestParameterizedHostTestClient
+
+"""
+The sample just shows how to use the method and may not run successfully.
+# PREREQUISITES
+    pip install AutoRestParameterizedHostTestClient
```

Comment:
we want the package name here, not the client name

Reply:
good point. I think I forget `|lower` in template files.